Pcap++/header/DpdkDevice.h

```diff
@@ -126,135 +131,143 @@ namespace pcpp
 
 	/**
 	 * @class DpdkDevice
-	 * Encapsulates a DPDK port and enables receiving and sending packets using DPDK as well as getting interface info & status, packet
-	 * statistics, etc. This class has no public c'tor as it's constructed by DpdkDeviceList during initialization.<BR>
+	 * Encapsulates a DPDK port and enables receiving and sending packets using DPDK as well as getting interface info &
+	 * status, packet statistics, etc. This class has no public c'tor as it's constructed by DpdkDeviceList during
+	 * initialization.<BR>
 	 *
-	 * __RX/TX queues__: modern NICs provide hardware load-balancing for packets. This means that each packet received by the NIC is hashed
-	 * by one or more parameter (IP address, port, etc.) and goes into one of several RX queues provided by the NIC. This enables
-	 * applications to work in a multi-core environment where each core can read packets from different RX queue(s). Same goes for TX
-	 * queues: it's possible to write packets to different TX queues and the NIC is taking care of sending them to the network.
-	 * Different NICs provide different number of RX and TX queues. DPDK supports this capability and enables the user to open the
-	 * DPDK port (DpdkDevice) with a single or multiple RX and TX queues. When receiving packets the user can decide from which RX queue
-	 * to read from, and when transmitting packets the user can decide to which TX queue to send them to. RX/TX queues are configured
+	 * __RX/TX queues__: modern NICs provide hardware load-balancing for packets. This means that each packet received
+	 * by the NIC is hashed by one or more parameter (IP address, port, etc.) and goes into one of several RX queues
+	 * provided by the NIC. This enables applications to work in a multi-core environment where each core can read
+	 * packets from different RX queue(s). Same goes for TX queues: it's possible to write packets to different TX
+	 * queues and the NIC is taking care of sending them to the network. Different NICs provide different number of RX
+	 * and TX queues. DPDK supports this capability and enables the user to open the DPDK port (DpdkDevice) with a
+	 * single or multiple RX and TX queues. When receiving packets the user can decide from which RX queue to read from,
+	 * and when transmitting packets the user can decide to which TX queue to send them to. RX/TX queues are configured
 	 * when opening the DpdkDevice (see openMultiQueues())<BR>
 	 *
 	 * __Capturing packets__: there are two ways to capture packets using DpdkDevice:
-	 *    - using worker threads (see DpdkDeviceList#startDpdkWorkerThreads() ). When using this method the worker should use the
-	 *      DpdkDevice#receivePackets() methods to get packets from the DpdkDevice
+	 *    - using worker threads (see DpdkDeviceList#startDpdkWorkerThreads() ). When using this method the worker
+	 * should use the DpdkDevice#receivePackets() methods to get packets from the DpdkDevice
```

Comment:
ditto: let's add trailing spaces to match alignment?

Reply:
Should be fixed in bf2631d